README.md

```diff
@@ -1,66 +1,114 @@
+In Maker's linguo, a spell is a bespoke smart contract to execute authorized actions in Maker Protocol on behalf on
+Maker Governance.
 
-## Documentation
+Since most contracts in Maker Protocol follow a [simple, battle-tested authorization scheme][auth], with an "all or
+nothing" approach, it means that every spell has _root_ access to every single of its components.
 
-https://book.getfoundry.sh/
+[auth]: https://github.com/makerdao/pe-checklists/blob/master/core/standards.md#permissions
 
-## Usage
+In order to mitigate the risks associated with that design decision, the spell process is quite "heavy", where
+multiple trusted parties are involved, and with comprehensive [checklists][spell-checklists] that must be strictly
+followed.
 
-### Build
+[spell-checklists]: https://github.com/makerdao/pe-checklists/tree/master/spell
 
-```shell
-$ forge build
-```
+With all the complexity and coordination effort, it is not a surprise that it takes a long time for a spell to be
+successfully crafted, reviewed and handed over to Maker Governance. As per the [current process][spell-schedule], with
+the involvement of at least 3 engineers from the different EAs in the Spell Team, not to mention the Governance
+Facilitators and other key stakeholders, it takes at least 8 working days to deliver a regular spell.
 
-### Test
+[spell-schedule]: https://github.com/makerdao/pe-checklists/blob/master/spell/spell-crafter-mainnet-workflow.md#spell-coordination-schedule
 
-```shell
-$ forge test
-```
+For emergency spells, historically the agreed SLA had been 24h. That was somehow possible when there was a single
+tight-knit team working on spells, however can be specially challenging with a more decentralized workforce, which is
+scattered around the world. Even if it was still possible to meet that SLA, in some situations 24h might be too much
+time.
 
-### Format
+This repository contains a couple of different spells performing emergency actions that can be pre-deployed to allow
+MakerDAO Governance to quickly respond to incidents, without the need for dedicated engineers to chime in and craft a
+bespoke spell in record time.
 
-```shell
-$ forge fmt
-```
+## Deployments
 
-### Gas Snapshots
+TBD.
 
-```shell
-$ forge snapshot
-```
+## Implemented Actions
 
-### Anvil
+| Description        | Single ilk         | Universal          |
+| :----------        | :--------:         | :-------:          |
+| Wipe `AutoLine`    | :white_check_mark: | :white_check_mark: |
+| Set `Clip` breaker | :white_check_mark: | :white_check_mark: |
+| Disable `DDM`      | :white_check_mark: | :x:                |
+| Stop `OSM`         | :white_check_mark: | :white_check_mark: |
 
-```shell
-$ anvil
-```
+### Wipe `AutoLine`
 
-### Deploy
+No further debt can be generated from an ilk which is wiped from `AutoLine`. It also prevents the debt ceiling
+(`line`) for the affected ilk from being changed without Governance interference.
```

Comment:
I think it would be great to link 1) contract you're talking about 2) mom contract

(here, but also in every other section)

Reply:
Wouldn't it be better to maybe reference the chainlog key instead?
Contracts can be updated, making this document stale.